src/blocks/RulesBlock.tsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+
+function RulesModuleHeader() {
+  return (
+    <div className="rules-header">
+      <h3>📌 Pinned Rules</h3>
+      <a href="#" className="rules-link">
+        See all
+      </a>
+      <style jsx>{`
+        .rules-header {
+          display: flex;
+          justify-content: space-between;
+          align-items: center;
+          padding: 6px;
+          height: 1.2em;
+        }
+        .rules-link {
+          color: #888;
+          text-decoration: none;
+          height: 1.2em;
+          padding: 0px 6px;
+        }
+      `}</style>
+    </div>
+  );
+}
+
+function RuleDisplay({ rule }) {
+  return (
+    <li key={rule.index}>
```

Comment:
FYI you don't need key here, it's only needed when rendered directly in the loop (so later on when you do `<RuleDisplay rule={rule} key={rule.title} />`)